resources/assets/components/utilities/SitewideBanner/SitewideBanner.js

```diff
@@ -36,13 +51,25 @@ const isExcludedPath = pathname => {
 
 const SitewideBanner = props => {
   const userId = getUserId();
+  const campaignId = campaign ? parseInt(campaign.campaignId, 10) : null;
+
   const options = { variables: { userId }, skip: !userId };
   const { data, loading } = useQuery(VOTER_REGISTRATION_STATUS, options);
-
   const unregistered = unregisteredStatuses.includes(
     get(data, 'user.voterRegistrationStatus'),
   );
 
+  const { data: campaignData, loading: campaignLoading } = useQuery(
+    CAMPAIGN_GROUPTYPE_QUERY,
+    {
+      variables: {
+        campaignId,
+      },
+      skip: !campaignId,
+    },
+  );
+  const campaignGroupTypeId = get(campaignData, 'campaign.groupType.id', null);
```

Comment:
I wonder if for readability below (that conditional is getting hefty lol), we can rename this to `isGroupCampaign` or something and render this as a Boolean:
```suggestion
  const isGroupCampaign = !!get(campaignData, 'campaign.groupType.id');
```